postgresdb/WarsawTransportState.sql

```diff
@@ -2,13 +2,14 @@ CREATE SCHEMA "dbo";
 
 CREATE TABLE "WarsawTransportState"."dbo"."stops" (
   "id" int PRIMARY KEY,
-  "full_name" int,
-  "longitude" double precision,
-  "latitude" double precision,
+  "full_name" varchar,
+  "stop_longitude" double precision,
+  "stop_latitude" double precision,
   "street" varchar,
   "unit" varchar,
   "post" varchar,
-  "is_depot" int
+  "is_depot" int,
```

Comment:
It's probably the result of the mistake in the diagram - but I believe it should be `boolean` instead `int`